tests/testing_common.py

```diff
@@ -380,7 +413,7 @@ def _test_training_layer_indexing(self, model_id, config_cls, config_kwargs):
             model.save_pretrained(tmp_dirname)
 
             model_from_pretrained = self.transformers_class.from_pretrained(model_id)
-            model_from_pretrained = PeftModel.from_pretrained(model_from_pretrained, tmp_dirname)
+            model_from_pretrained = PeftModel.from_pretrained(model_from_pretrained, tmp_dirname).to(self.torch_device)
```

Comment:
was adding `.to(self.torch_device)` necessary?

Reply:
Yes it is something that I forgot to add when I made the CI PR